admin-dev/themes/new-theme/scss/components/layout/_header_toolbar.scss

```diff
@@ -4,7 +4,7 @@
   top: $size-header-height;
   left: $size-navbar-width;
   right: 0;
-  z-index: 1100; // 1070 is the highest z-index in bootstrap v4.0.0 (help popover)
+  z-index: 10;
```

Comment:
We should find out the Z-index of modals and set it just below it instead.

Reply:
Okay, so this should be done is the ui-kit?

Reply:
If we go for this, should this be done is the ui-kit?

It seems the tooltips seem to be the only thing being misplaced regarding the z-index. I would prefer keeping the current issue (or only update them?) instead of modyfing all the back-office elements.

Reply:
The z-index for modals is 1050 and their backdrops 1040. So any value lower than that would be an improvement.

Reply:
Whatever we chose (1040 or 10), the modal is properly on top, but we go back to the original issue.

![capture du 2018-11-01 10-02-28](https://user-images.githubusercontent.com/6768917/47845605-64881f00-ddbd-11e8-9064-eb892f0b2035.png)

Having to modify the z-index header, modal... seems to be a bad idea IMO.

Reply:
Can't you just copy the original Bootstrap 4 values?